commands/instances.go

```diff
@@ -451,6 +458,19 @@ func Init(req *rpc.InitRequest, responseCallback func(r *rpc.InitResponse)) erro
 	return nil
 }
 
+func areDiscoveriesRunning(pm *packagemanager.PackageManager) bool {
+	if pm == nil {
+		return false
+	}
+	exp, release := pm.NewExplorer()
+	defer release()
+
+	if exp.DiscoveryManager() != nil && exp.DiscoveryManager().AreDiscoveriesRunning() {
```

Comment:
please let me know if there's a simpler way to get the status

Reply:
It is not needed, see my other comments.